### ISSUE TITLE
Newly created folder is not shown

### ISSUE THREAD

**Issue by AJamesPhillips** [inrupt/pod-browser#333]
Steps to recreate:
1. Using code, create a new file nested under a folder at the top level, e.g. pod.inrupt.com/<user name>/tmp/newfile.ttl
2. Refresh the pod browser for your pod multiple times
3. Observe no new directory is present 
4. Using code, create a new file at the top level, e.g. pod.inrupt.com/<user name>/newfile2.ttl
5. Refresh the pod browser for your pod once
6. Observe the newfile2 and the new directory with its file is now shown

Please see [this video](https://user-images.githubusercontent.com/526509/130832755-9bc04795-26c7-484a-9158-8b4b3fd7acae.mp4) for full demonstration.

Code used is here: https://github.com/AJamesPhillips/solid-bugs/tree/23f86ba

**Comment by ThisIsMissEm** [comment]
I don't think this is a pod-browser issue, it sounds like something isn't being stored or cached right; i.e., if you did `getSolidDataset` in your app, I think you'd see the same in that data as what pod-browser shows.

But you may want to retest just to see if it's still an issue.
